src/docfx/template/MustacheTemplate.cs

```diff
@@ -37,11 +37,19 @@ RendererSettingsBuilder UseJson(RendererSettingsBuilder settings)
             }
         }
 
-        public string Render(string templateName, JToken model)
+        public string Render(string templateFileName, JToken model)
```

Comment:
the mustache template wasn't run, based on diff, can you add a spec test for mustache template?

Reply:
added a spec test to verify SDP pipeline